src/app/_components/countdown/countdown-wrapper.tsx

```diff
@@ -0,0 +1,13 @@
+import { serverTRPC } from "@/app/_trpc/server";
+import Countdown from "./countdown";
+import CountdownComplete from "./countdown-complete";
+
+export default async function CountdownWrapper() {
+  const { eventStartTime, timeRemaining } =
+    await serverTRPC.countdown.get_time_remaining.query();
+
+  if (timeRemaining <= -43200000) return <CountdownComplete />;
```

Comment:
Why can't this if statement return Countdown complete as part of the countdown component below?

Reply:
You're right. I thought it'd take more time to determine whether the event has completed since it has to run through calculations in 'countdown', but that doesn't seem to be the case. I tested your suggestion and everything checks out.